HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/Info.plist

```diff
@@ -60,5 +60,12 @@
 		<string>ginipay-bank</string>
 		<string>ginipay-ginibank</string>
 	</array>
+	<key>UIApplicationSceneManifest</key>
```

Comment:
@ValentinaIancu-Gini  do we still need this change?

Reply:
yes, otherwise there is a warning in the console that this is missing.